src/app.js

```diff
@@ -1,12 +1,16 @@
 import express from 'express';
 import cookieParser from 'cookie-parser';
 import bodyParser from 'body-parser';
+import morgan from 'morgan';
 import routes from './routes';
 import { passport } from './config/passport';
 import { queryParser, setCoockies, checkToken } from './middlewares/middlewares';
 
 const app = express();
 
+morgan.token('id', req => req.id);
+app.use(morgan('combined'));
+
 app.use(queryParser());
 app.use(cookieParser(), setCoockies());
 app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
There is from Express v4 `bodyParser` libraty doesn't need because the one already is situated in Express library.
```
app.use(express.urlencode({ extenred: true }));
app.use(express.json());
```